### DISCUSSION TITLE
друкування рахунків, актів, замовлень

### DISCUSSION THREAD

**Discussion by MusienkoAnton** [leon-mbs/zstore#664] [Q&A] [answered]
Доброго дня
У демоверсії на сайті не знайшов можливості друкувати рахунки, замовлення. Підкажіть де це можно зробити?

**Comment by leon-mbs** [comment, marked as answer]
Здравствуйте, MusienkoAnton.
 
наприклад в  журганалi документiв клiкаете перегляд (по глзiку)
там  iконки  друку

Вы писали 23 December, 2023, 15:49:17:

> Доброго дня
> У демоверсії на сайті не знайшов можливості друкувати рахунки, замовлення. Підкажіть де це можно зробити?
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.

-- 
С уважением,
 gmail                         ***@***.***
